packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -602,7 +611,9 @@ export function TransferPanel() {
           receiveMessageTransactionHash: null,
           receiveMessageTimestamp: null
         }
-      })
+      }
+      const mutate = isDeposit ? setPendingDeposit : setPendingWithdrawal
+      mutate(newTransfer)
```

Comment:
I'd rename the generic mutate function to more contextual like - `setTransferInCache(newTransfer)`